lms/djangoapps/course_api/blocks/api.py

```diff
@@ -94,6 +104,8 @@ def get_blocks(
     # TODO: Remove this after REVE-52 lands and old-mobile-app traffic falls to < 5% of mobile traffic
     if is_request_from_mobile_app(request):
```

Comment:
There's no reason for `VideoBlockURLTransformer` to be called only for mobile requests, right?  I would think that this translation should happen even when called by other clients, right?

Reply:
Actually, yes. The rewrite in the web app, for the video blocks, is already happening in [video_module](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/video_module/video_module.py#L329-L333). The sole reason for this transformer is to do the same with both mobile applications. Since the data is being retrieved from the course API from the mobile apps, adding this transformer takes care of the rewrite.

Reply:
In that case, we definitely want to do this transformation for all requests, not just from mobile apps.  Eventually, we will use this Blocks API for all clients, including Web clients as well.  And we want all callers that may exist today (outside of mobile) to have the translated video URLs.

Reply:
Ok. Let me discuss internally. It makes sense to do this with a long term perspective in mind.